packages/block-editor/src/components/inserter/style.scss

```
@@ -279,10 +279,9 @@ $block-inserter-tabs-height: 44px;
 	padding: $grid-unit-20;
 	overflow-y: auto;
 	height: 100%;
-}
-
-.block-editor-inserter__patterns-fill-space {
-	flex-grow: 1;
+	div[role="listitem"]:last-child {
```

Maybe we could add an inline comment to explain what this rule does?  E.g

```suggestion
    // Push the listitem wrapping the "explore" button to the bottom of the panel
	div[role="listitem"]:last-child {
```